app/client/templates/autoform/imageUpload.js

```diff
@@ -0,0 +1,12 @@
+Template.imageUpload.rendered = function () {
+};
+
+Template.imageUpload.events({
+  'change .image-upload-input': function(e) {
+    e.preventDefault();
+  },
+
+  'click .image-upload-trigger': function(e) {
+    e.preventDefault();
```

Comment:
Is this waiting for upload logic? If so probably commented placeholder or something would help readability